src/sass/app_2/modules/trading.scss

```diff
@@ -632,4 +674,18 @@
             }
         }
     }
+    fieldset {
+        &.barriers {
+            .input-field {
+                &.multiple {
+                    .up {
+                        background-color: $COLOR_WHITE;
```

Comment:
You can combine `up` and `down` classes here since they're both the same `background-color`.